### PR TITLE
French Translation of sequential edit article

### PR DESCRIPTION
**Pull request by Tipiakus**
New attempt to PR the French translation of the sequential edit article

### CODE REVIEW

**Comment by Tipiakus** [comment]
Redimensionnement des images.

File: content/md/productivity/sequential-edit_fr.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# Introduction
+
+La modification séquentielle permet d’afficher les unes après les autres les pages Produit à partir d’une sélection de produits, sans passer par la grille des produits. Cela permet de gagner du temps en permettant la navigation directe entre les pages Produit des produits sélectionnés.
+
+Tout utilisateur disposant d'un accès au produit avec des autorisations de modification peut exécuter une modification séquentielle. Il n’existe aucun droit spécifique pour donner accès à cette fonctionnalité.
+
+
+# Comment faire ?
+
+Pour lancer la modification séquentielle du produit :
```
Comment (NolwennP) [inline content/md/productivity/sequential-edit_fr.md]:
```suggestion
Pour lancer la modification séquentielle de produits :
```

File: content/md/productivity/sequential-edit_fr.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+1. Recherchez des produits dans la grille des produits avec les filtres appropriés,
+2. Sélectionnez les produits sur lesquels vous souhaitez effectuer l'opération via l'outil de sélection :
+     - `Tous` pour tous les produits enregistrés,
```
Comment (NolwennP) [inline content/md/productivity/sequential-edit_fr.md]:
```suggestion
     - `Tous` pour tous les produits de la grille,
```

File: content/md/productivity/sequential-edit_fr.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+La barre de progression indique le nombre de produits qui ont déjà été modifiés par rapport au nombre total de produits sélectionnés. La barre de progression est entourée de deux liens à gauche et à droite.
+
+![image](Products_SequentialEditProgressBar2_fr.png)
+
+Le lien de gauche vous permet de revenir au produit précédent, le lien de droite pour passer au produit suivant.
+
+::: ee
+Les produits avec les droits d'accès en lecture seule seront seulement affichés et ne seront pas modifiables.
+
+<!--
```
Comment (NolwennP) [inline content/md/productivity/sequential-edit_fr.md]: You can remove the whole block, as for now, we couldn't reproduce this behavior.

Comment (Tipiakus) [reply content/md/productivity/sequential-edit_fr.md]: Done, I removed this part

**Comment by Tipiakus** [comment]
Pour résoudre le conflit il ne faut pas garder les visuels de cette Pull Request

**Pull request merged**